### PR TITLE
[#1878] Create ordering API endpoints

### LINKED ISSUE #1878: Create ordering API controller and OpenAPI specs
https://docs.cyfronet.pl/display/FID/API+for+resource+orders#APIforresourceorders-OrderingAPI

Create a controller(s?) with the following stubbed endpoints, described in the doc above:
- GET /oms/{oms_id}/events
- GET /oms/{oms_id}/projects
- GET /oms/{oms_id}/projects/{p_id}
- PATCH /oms/{oms_id}/projects/{p_id}
- GET /oms/{oms_id}/project_items
- GET /oms/{oms_id}/project_items/{pi_id}
- PATCH /oms/{oms_id}/project_items/{pi_id}
- GET /oms/{oms_id}/messages
- POST /oms/{oms_id}/messages
- PATCH /oms/{oms_id}/messages/{m_id}

Expose them as OpenAPI.

### PR DESCRIPTION
**Pull request by wujuu**
Closes #1878 

**What has been changed**:
- Add stub endpoints
- Add stub endpoint rswag tests
- Add json schemas to endpoints
- Add pundit authorization stubs
- Remove OFFERS_API_DISABLED flag
- Remove api_docs data_administrator authorization

**For reviewers**:
Go to  `/api_docs/swagger`, select "Ordering API" from the dropdown menu in the top right corner, you don't have to be logged in.

### CODE REVIEW

File: app/controllers/api/v1/oms/projects/project_items_controller.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            "type": "rejected"
+          },
+          "attributes": {
+            "reason_for_access": "",
+            "country_of_origin": "",
+            "customer_typology": "",
+          },
+          "delivery_details": {
+            "custom": "fields"
+          },
```
Comment (jswk) [inline app/controllers/api/v1/oms/projects/project_items_controller.rb:22]: These should be kept only on the MP side.

File: app/controllers/api/v1/oms/projects/project_items_controller.rb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            "value": "<status>",
+            "type": "waiting_for_response"
+          },
+          "attributes": {
+            "etc1": "",
+            "etc3": "",
+          },
+          "delivery_details": {
+            "custom1": "fields"
+          },
```
Comment (jswk) [inline app/controllers/api/v1/oms/projects/project_items_controller.rb:40]: These should be kept only on the MP side.

File: app/controllers/api/v1/oms/projects/project_items_controller.rb [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        "type": "rejected"
+      },
+      "attributes": {
+        "reason_for_access": "",
+        "country_of_origin": "",
+        "customer_typology": "",
+      },
+      "delivery_details": {
+        "custom": "fields"
+      },
```
Comment (jswk) [inline app/controllers/api/v1/oms/projects/project_items_controller.rb:65]: These should be kept only on the MP side.

File: spec/requests/api/v1/oms/project/project_items_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+              "type": "rejected"
+            },
+            "attributes": {
+              "reason_for_access": "",
+              "country_of_origin": "",
+              "customer_typology": "",
+            },
+            "delivery_details": {
+              "custom": "fields"
+            },
```
Comment (jswk) [inline spec/requests/api/v1/oms/project/project_items_controller_spec.rb:83]: These should be kept only on the MP side.

File: swagger/v1/ordering/project/project_item/project_item_read.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+      "properties":{
+        "value": {"type": "string"},
+        "type": {"type": "string",
+          "enum": ["rejected", "waiting_for_response", "registered", "in_progress", "ready", "closed", "approved"]
+        }
+      },
+      "required": ["value", "type"]
+    },
+    "attributes": {"type": "object"},
+    "delivery_details": {"type": "object"},
```
Comment (jswk) [inline swagger/v1/ordering/project/project_item/project_item_read.json:17]: These should be kept only on the MP side.

**Review by jswk — APPROVED** [review]

File: app/controllers/api/v1/oms/projects/project_items_controller.rb (hunk 1 above)
Comment (bwilk) [reply app/controllers/api/v1/oms/projects/project_items_controller.rb:22]: Hmm. I'd say we need to discuss this. I'm slightly biased on this one

**Pull request merged**